pkgs/servers/squid/default.nix

```diff
@@ -9,6 +9,14 @@ stdenv.mkDerivation rec {
     sha256 = "1n4f55g56b11qz4fazrnvgzx5wp6b6637c4qkbd1lrjwwqibchgx";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
I think it'd probably be nicer to keep the patch in `nixpkgs` itself, just to minimise network access dependencies

Reply:
Yeah I'm kind of on the fence with this one - but the general policy these days seems to be only add the patch to nixpkgs if it's unavoidable. This is made less painful because once packages make it into nixpkgs the nix cache grabs a copy of all their http fetches. I can't remember the last time my machine had to fetch a patch from a third party url except when I'm reviewing unmerged work.